backend/multiple_level_preprocess.py

```diff
@@ -43,7 +36,7 @@ class MultipleLevelPreprocess:
     in order to keep constant loading time.
     To manage the knowledge of each level, a metadata.json file is genereted for the entire
     file and each slice, including names, start time of slice, start, end, and frequency, etc.
-    Metadata is saved in json format, and one for raw data, one for each level.
+    Metadata is saved in json format, and one for raw data, one for each level in each strategy.
```

Comment:
Rephrase this line to something like:

All metadata files are in json format. One is the high level description about the raw data and downsampled data in each level. The rest of the metadata files describe the detail of downsampled data in each level in each strategy.